src/ioc-react/mapDependenciesToProps.ts

```diff
@@ -0,0 +1,17 @@
+import { Container } from '../ioc';
+
+/**
+ * @ignore
```

Comment:
what is @ignore?

Reply:
@jhoguet its jsdoc (or maybe tsdoc) to prevent this from being included in the generated documentation